thumbservice/common.py

```diff
@@ -12,13 +12,14 @@ class Settings:
     def __init__(self, settings=None):
         self._settings = settings or {}
 
-        self.ARCHIVE_API = self.set_value('ARCHIVE_API', 'https://archive-api.lco.global/', True)
+        self.ARCHIVE_API = self.set_value('ARCHIVE_API', 'http://localhost/', True)
         self.TMP_DIR = self.set_value('TMP_DIR', '/tmp/', True)
-        self.BUCKET = self.set_value('AWS_S3_BUCKET', 'lcogtthumbnails')
+        self.BUCKET = self.set_value('AWS_S3_BUCKET', 'changeme')
         self.AWS_ACCESS_KEY_ID = self.set_value('AWS_ACCESS_KEY_ID', 'changeme')
         self.AWS_SECRET_ACCESS_KEY = self.set_value('AWS_SECRET_ACCESS_KEY', 'changeme')
-        # Using `None` for `STORAGE_URL` will connect to AWS
-        self.STORAGE_URL = self.set_value('STORAGE_URL', None)
+        self.REQUIRED_FRAME_VALIDATION_KEYS = self.get_tuple_from_environment('REQUIRED_FRAME_VALIDATION_KEYS', 'configuration_type,request_id,filename')
```

Comment:
I figured I'd just leave these as sensible defaults, but I could also do away with them and set them in our helm chart.